lib/kafo/kafo_configure.rb

```diff
@@ -523,7 +523,7 @@ def run_installation
               method, message = log_parser.parse(line)
               progress_log(method, message, logger)
 
-              if (output = line.match(/(.+\]): Starting to evaluate the resource( \((?<count>\d+) of (?<total>\d+)\))?/))
+              if (output = line.match(/(.+\]): Starting to evaluate the resource( \((\d+) of (\d+)\))?/))
```

Comment:
What you're removing are called named captures and are used on the line below. https://www.leighhalliday.com/named-captures-ruby-regular-expressions explains them. They are extremely useful in complex regular expressions because the numbers are very easy to mess up.

https://docs.rubocop.org/rubocop/cops_lint.html#lintmixedregexpcapturetypes explains the cop. The way to deal with it is to use `?:` for groups you discard
```suggestion
              if (output = line.match(/(?:.+\]): Starting to evaluate the resource( \((?<count>\d+) of (?<total>\d+)\))?/))
```